hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/utils/CheckedConsumer2E.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.mono.state.virtual.utils;
+
+public interface CheckedConsumer2E<T, U, E extends Exception> {
```

Comment:
There is an existing class `CheckedConsumer2` in package `com.hedera.node.app.service.mono.state.virtual.utils`. I just copied it. However, I agree it would be better named `CheckedBiConsumer`.

Update: I found that after these PR, the old interfaces `CheckedConsumer`, `CheckedConsumer2`, and `CheckedSupplier` are no longer used. I'll remove them.